src/OneExplorer/OneExplorer.ts

```diff
@@ -559,6 +559,26 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
       vscode.commands.registerCommand("one.explorer.runCfg", (node: Node) => {
         vscode.commands.executeCommand("one.toolchain.runCfg", node.uri.fsPath);
       }),
+      vscode.commands.registerCommand(
+        "one.explorer.runSingleSelectedCfg",
+        () => {
+          let selectedCfg = provider.getSelectedCfg();
+          if (selectedCfg && selectedCfg.length === 1) {
+            vscode.commands.executeCommand(
+              "one.toolchain.runCfg",
+              selectedCfg[0].uri.fsPath
+            );
+          } else if (!selectedCfg || selectedCfg.length === 0) {
+            // TODO: handle for none selection
+            vscode.window.showErrorMessage("No selected cfg file to run.");
+          } else if (selectedCfg.length !== 1) {
+            // TODO: handle for multiple selection
+            vscode.window.showErrorMessage(
+              "No single selected cfg file to run."
```

Comment:
How about this message instead? 
```suggestion
              "Cannot run multiple cfg files. Please select a single cfg file."
```